docs/SRS-Volere/SRS.tex

```diff
@@ -60,21 +60,131 @@ \subsection{Goals of the Project}
+        and other tasks. They will have to be able to achieve all their needed requirements through
+        the applications user interface, and will be a critical stakeholder to receive feedback
+        from through user testing.
+    \end{itemize}
+    \item Facilities co-op students
+    \begin{itemize}
+        \item[-] Facilities co-op students will be using the application to physically authenticate on site
+        when performing inspections, scoping work, and verifying completion of work.
+    \end{itemize}
+    \item Facilities contractors
+    \begin{itemize}
+        \item[-] Many contractors access pumping stations daily. They will interact with
+        the application each time they perform facilities related work at the station. Interactions
+        would include uploading and receiving documentation, and authenticating their presence on site 
+        to verify completion of work.
+        Contractors come from a wide range of backgrounds depending on their trade, but are typically skilled 
+        workers in a particular field and are of working age (18 - 65 years old).
+    \end{itemize}
+    \item Facilities sub-contractors
+    \begin{itemize}
+        \item[-] Many contractors employ sub-contractors as part of their work.
+        While these subcontractors work for the contractor who hired them, they will still require access
+        to the application to authenticate on site and access site information. Their roles and interactions
+        will be similar to contractors, except they will be much less familiar with the application
+        and may only use it for a short duration as they are not regular workers.
+    \end{itemize}
+\end{itemize}
 \subsection{Personas}
-\lips
+
+\begin{itemize}
+    \item Greg: A facilities manager
+    \begin{itemize}
+        \item[-] Greg has a few tasks to complete this morning at work. He sees that another department
+        has requested compliance documentation on crane inspections for the ministry, so he signs onto
+        SyncMaster to retrieve the current version of these documents for each of these sites. While
+        on the application, he receives a notification that a contractor he manages authenticated at site
+        to complete a work order, but the application identified they had not completed their
+        health and safety training. Greg prompts the system to issue this training to them, and runs a report
+        on this contractor to see if any other employees of theirs require their training soon.
+    \end{itemize}
+    \item Nancy: A newly onboarded contractor
+    \begin{itemize}
+        \item[-] Nancy is a plumber who is servicing her first work order after being hired by "Worldly Plumbing".
+        Her company has notified her of the application she must use to authenticate for the work.
+        When she authenticates, she is notified that the device she is servicing is located within a confined space
```

Comment:
I think this and a couple others go over the 80 char line length limit https://github.com/Spitgranger/capstone/blob/main/docs/Checklists/Writing-Checklist.pdf

Reply:
thanks,

just found a way to have vim have a char limit indicator, use:
set colorcolumn=80

![image](https://github.com/user-attachments/assets/2e5b0d3f-a588-4e20-a674-af0f20d3ea96)

It looks like some of the lines provided by the template itself are over the character limit

Reply:
fixed.